test/index.js

```diff
@@ -128,6 +129,106 @@ test("constructs and calls callback without throwing", function (assert) {
     assert.end()
 })
 
+test("[func] xhr[method] get, put, post, patch", function (assert) {
+  var i = 0
+  methods = ["get", "put", "post", "patch"]
+  forEach(methods, function (method) {
+      xhr[method]({
+          uri: "https://httpbin.org/" + method
```

Comment:
hmm. worked fine in Chrome for me:

![screenshot 2015-10-23 14 42 05](https://cloud.githubusercontent.com/assets/755844/10705638/480cbf6a-7994-11e5-97e0-57ec5b39c5a0.png)